test/integration/render-tests/hillshade-accent-color/literal/style.json

```diff
@@ -0,0 +1,38 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "height": 256,
+      "width": 256
+    }
+  },
+  "center": [-113.26903, 35.9654],
+  "zoom": 11,
+  "sources": {
+    "source": {
+      "type": "raster-dem",
+      "tiles": [
+        "local://tiles/{z}-{x}-{y}.terrain.png"
+      ],
+      "maxzoom": 15,
+      "tileSize": 256
+    }
+  },
+  "layers": [
+    {
+      "id": "background",
+      "type": "background",
+      "paint": {
+        "background-color": "white"
+      }
+    },
+    {
+      "id": "hillshade",
+      "type": "hillshade",
+      "source": "source",
+      "paint": {
+        "hillshade-accent-color": "purple"
```

Comment:
Just for fun I tried changing this to "red" and discovered that, while I can see the difference in the image, it still passed the render test -- I guess the difference was within the threshold.  Is there an image/location we could use for the hillshade-accent  that would have more pronounced accents so that they're more likely to catch subtle regressions?